com.unity.render-pipelines.universal/Editor/GlobalSettings/SerializedUniversalRenderPipelineGlobalSettings.cs

```diff
@@ -20,6 +20,9 @@ class SerializedUniversalRenderPipelineGlobalSettings
         public SerializedProperty lightLayerName7;
 
         public SerializedProperty supportRuntimeDebugDisplay;
+        public SerializedProperty staticVolumeProfile;
+        public SerializedProperty stripOffVariants;
+        public SerializedProperty stripBuiltinShaders;
```

Comment:
With @TheoWong-pixel we decide probably not include stripBuiltinShaders as users can easily shot themselves with it. As currently there is no trivial way to report to user if they use builtin shader that would be stripped by this option. I am open for opinions

Reply:
I agree, if off by default then I think we should give users this control. But we should probably have some docs about the implications of it.